### ISSUE TITLE
What should I do if the synthetic image in target is similar the real image in source?

### ISSUE THREAD

**Issue by John1231983** [junyanz/pytorch-CycleGAN-and-pix2pix#618]
Hello, I am using the cycleGAN for making a synthetic image from source domain A to target domain B. After training 10.000 epochs from dataset with 10.000 images. I used U-net generator with ngf 64 and discriminator ndf 64. The size of the last layer in D network is 7x7. 

The problem is that the synthetic image in the target domain similar to the real image in the source domain. What is happening in my problem?  Thanks so much

**Comment by junyanz** [comment]
I recommend that you use the Resnet generator for CycleGAN training. What's the problem? The more similar two domains are, the easier the model training will be.

**Comment by John1231983** [comment]
Sorry I may write the wrong thing so you may misunderstand. I mean that the result of Unet gives the synthetic image is too similar to the input source image. I mean that the trained unet generator looks feeds directly the input to the output :(. so nothing to learning in down-sampling path. I tried the resnet before but training speed too slow although learning parameter less than Unet

**Comment by junyanz** [comment]
I recommend that you use ResNet for CycleGAN, as U-Net doesn't work as well as ResNet in the CycleGAN model. You can use a smaller model `resnet_6blocks` is speed is a concern. It takes some time for a model to learn the mapping. If you like bigger changes, you can also decrease the cycle consistency loss.

**Comment by John1231983** [comment]
Thanks for your point. I tried both unet and resnet and result are the same. It does not solve my problem. The mapping from source to target are identical. It means the result of the synthetic image similar to the real image. Do you know what is the name of the issue?

**Comment by junyanz** [comment]
I don't know the name of this issue. Maybe try a smaller cycle loss.
